### ISSUE TITLE
License

### ISSUE THREAD

**Issue by brandonkal** [vtex/styleguide#625]
This looks quite nice! I quite like the date picker, toast provider, and checkboxes. I notice there is no license. I would like the option to use some of these components and styles for an internal user interface. It is of course fine if you do not want to open source this, but in that case a short section in the readme saying so would be helpful.

If you do decide to license this, I have seen other style guides effectively use a permissive license with the added condition of not using the components to build a competing product or anything that would otherwise cause confusion. So that is an idea if that was a concern.

Thank you.

**Comment by klzns** [comment]
Hi @brandonkal, thanks! 

> If you do decide to license this, I have seen other style guides effectively use a permissive license with the added condition of not using the components to build a competing product or anything that would otherwise cause confusion.

I like this idea! Do you have a link to it?

**Comment by brandonkal** [comment]
Yes. Such a clause is found in Shopify [Polaris design guidelines](https://polaris.shopify.com/legal/license) condition 2.

Their license actually goes further than what I suggested, as it states that Polaris can only be used for things that integrate with their platform. I believe that is a shame but I do like the idea of putting some restrictions so someone isn't free to clone the product itself using the same design.  Their component license is [much shorter](https://github.com/Shopify/polaris-react/blob/master/LICENSE.md).

On the flip side, Material Design is permissive MIT, and Adobe's Spectrum is Apache. If someone did try to build a Google clone, they could have a strong case on trade dress infringement, so a more restrictive license technically isn't required at all.

Even if such a restriction were not in place, I would still tweak color variables etc to differentiate. Still, it is a nice idea in theory to make non-competition an explicit license requirement. As it is now, I think too many sites look Google/Androidish so it would be refreshing to have more options. I believe adding such a clause would make it easier for us to share design work so we don't have to keep reinventing the wheel here.

If it was desired to place an explicit restriction on colors, it would be useful to place that restriction only on public apps to make quick prototyping possible:

"You are free to use these components as provided for the use of prototypes, testing, and internal tools. You may also use these components for public applications provided that you or your public applications are not a direct competitor to the LICENSOR and you take appropriate steps to ensure visual distinction such as changing brand color variables."

Thanks.

**Comment by ericreis** [comment]
I really think that this issue should be addressed ASAP! 

@klzns Do you have any idea of someone who could help us define our license model?
